Doc/library/zipfile.rst

```diff
@@ -79,9 +79,9 @@ The module defines the following items:
    of the last modification to the file; the fields are described in section
    :ref:`zipinfo-objects`.
 
-   .. versionadded:: 3.13
-      A public ``.compress_level`` attribute has been added to expose the
-      formerly protected ``._compresslevel``.  The older protected name
+   .. versionchanged:: 3.13
+      A public :attr:`compress_level` attribute has been added to expose the
```

Comment:
Instead of re-adding zipfile.rst to .nitignore, just ignore the link. We won't forget to fix this, @CAM-Gerlach :)

```suggestion
      A public :attr:`!compress_level` attribute has been added to expose the
```

Reply:
Or just revert that change, since explicitly silencing the resolution implies that it _shouldn't_ resolve as opposed to just isn't addressed yet.

```suggestion
      A public ``compress_level`` attribute has been added to expose the
```

Reply:
Err, too late now I guess...doesn't make much of a difference so long as we fix it promptly.